spec/unit/coinbase/transfer_spec.rb

```diff
@@ -69,6 +69,34 @@
                                      'transaction_hash' => transaction_hash
                                    })
   end
+  let(:complete_broadcast_model) do
+    Coinbase::Client::Transfer.new({
+                                     'network_id' => network_id,
+                                     'wallet_id' => wallet_id,
+                                     'address_id' => from_address_id,
+                                     'destination' => to_address_id,
+                                     'asset_id' => 'eth',
+                                     'amount' => amount.to_s,
+                                     'transfer_id' => transfer_id,
+                                     'status' => 'complete',
+                                     'unsigned_payload' => unsigned_payload,
+                                     'signed_payload' => signed_payload,
+                                     'transaction_hash' => transaction_hash
+                                   })
```

Comment:
```suggestion
    Coinbase::Client::Transfer.new(
      network_id: network_id,
      wallet_id: wallet_id,
      address_id: from_address_id,
      destination: to_address_id,
      ...
    )
```

We're adding a lot of unnecessary curly braces and stringify keys for these client model initializations.

Not blocking, but let's use the new paradigm going forwards as it's a lot more readable and standard post ruby 1.9 (which is decades old at this point 😂 )